TKGame/Level Editor Content/Stages/auto_saved_stage_data_8.json

```diff
@@ -0,0 +1,44 @@
+[
+  {
+    "X": 0,
+    "Y": 860,
+    "dataWidth": 1600,
+    "dataHeight": 50
+  },
+  {
+    "X": 0,
+    "Y": 0,
+    "dataWidth": 50,
+    "dataHeight": 650
+  },
+  {
+    "X": 1550,
+    "Y": 0,
+    "dataWidth": 50,
+    "dataHeight": 650
+  },
+  {
+    "X": 800,
+    "Y": 450,
+    "dataWidth": 250,
+    "dataHeight": 250
+  },
+  {
+    "X": 864,
+    "Y": 96,
+    "dataWidth": 608,
+    "dataHeight": 96
+  },
+  {
+    "X": 1376,
+    "Y": 384,
+    "dataWidth": 64,
+    "dataHeight": 128
+  },
+  {
+    "X": 448,
+    "Y": 64,
+    "dataWidth": 128,
+    "dataHeight": 96
+  }
+]
```

Comment:
Brother please the auto saves

Reply:
Make sure if you're exiting with ESC instead of closing the window with the X that you delete the autosaved stages so you don't inflate your PR

Reply:
I removed all the extra auto saves from my repo